tools/README.md

```diff
@@ -265,6 +265,50 @@ Compare Executor Information:
 +--------+-----------------+------------+-------------+-----------+------------+-------------+-------------+--------------+------------------+---------------+-------+-------+
 ```
 
+- Matching SQL IDs Across Applications:
+```
+Matching SQL IDs Across Applications:
++-----------------------+-----------------------+
+|app-20210329165943-0103|app-20210329170243-0018|
++-----------------------+-----------------------+
+|0                      |0                      |
```

Comment:
would it be helpful to also add time each took?  I guess we can always add more later.

Reply:
Ya I thought about that, but I didn't want to make it too complicated to start out with.